modules/fx/arrow-fx/src/main/kotlin/arrow/fx/extensions/QueueFunctor.kt

```diff
@@ -0,0 +1,27 @@
+package arrow.fx.extensions
+
+import arrow.Kind
+import arrow.extension
+import arrow.fx.Queue
+import arrow.fx.QueueOf
+import arrow.fx.QueuePartialOf
+import arrow.fx.fix
+import arrow.typeclasses.Functor
+import arrow.undocumented
+
+@extension
+@undocumented
+interface QueueFunctor<F, E> : Functor<QueuePartialOf<F, E>> {
+  fun functorF(): Functor<F>
+  override fun <B, C> QueueOf<F, E, B>.map(f: (B) -> C): Queue<F, E, C> =
```

Comment:
Nit: Why B to C instead of A to B?

Reply:
I had some issues with the names (#1905)and wasn't sure about the right solution. I will fix it!